systemtests/src/main/java/com/github/streamshub/systemtests/constants/ResourceKinds.java

```diff
@@ -6,37 +6,41 @@ private ResourceKinds() {}
-    public static final String DEPLOYMENT_CONFIG = "deploymentconfig";
-    public static final String POD = "pod";
-    public static final String SERVICE = "service";
-    public static final String SECRET = "secret";
-    public static final String CLUSTER_ROLE = "clusterrole";
-    public static final String CLUSTER_ROLE_BINDING = "clusterrolebinding";
-    public static final String ROLE_BINDING = "rolebinding";
-    public static final String SERVICE_ACCOUNT = "serviceaccount";
-    public static final String JOB = "job";
-    public static final String NAMESPACE = "namespace";
-    // ----------
-    // Openshift
-    // ----------
-    public static final String CATALOG_SOURCE = "catalogsource";
-    public static final String SUBSCRIPTION = "subscription";
-    public static final String INSTALL_PLAN = "installplan";
-    public static final String CLUSTER_SERVICE_VERSION = "clusterserviceversion";
-    public static final String OPERATOR_GROUP = "operatorgroup";
-    public static final String OPERATOR = "operator";
+    public static final String CUSTOM_RESOURCE_DEFINITION = "CustomResourceDefinition";
+    public static final String CONFIG_MAP = "ConfigMap";
+    public static final String DEPLOYMENT = "Deployment";
+    public static final String DEPLOYMENT_CONFIG = "DeploymentConfig";
+    public static final String POD = "Pod";
+    public static final String SERVICE = "Service";
+    public static final String SECRET = "Secret";
+    public static final String CLUSTER_ROLE = "ClusterRole";
+    public static final String CLUSTER_ROLE_BINDING = "ClusterRoleBinding";
+    public static final String ROLE_BINDING = "RoleBinding";
+    public static final String SERVICE_ACCOUNT = "ServiceAccount";
+    public static final String JOB = "Job";
+    public static final String NAMESPACE = "Namespace";
+
+    // ----------
+    // OpenShift
+    // ----------
+    public static final String CATALOG_SOURCE = "CatalogSource";
+    public static final String SUBSCRIPTION = "Subscription";
+    public static final String INSTALL_PLAN = "InstallPlan";
+    public static final String CLUSTER_SERVICE_VERSION = "ClusterServiceVersion";
+    public static final String OPERATOR_GROUP = "OperatorGroup";
+    public static final String OPERATOR = "Operator";
+
     // ----------
     // Kafka
     // ----------
-    public static final String KAFKA_TOPIC = "kafkatopic";
-    public static final String KAFKA = "kafka";
-    public static final String KAFKA_USER = "kafkauser";
-    public static final String KAFKA_NODE_POOL = "kafkanodepool";
+    public static final String KAFKA_TOPIC = "KafkaTopic";
+    public static final String KAFKA = "Kafka";
+    public static final String KAFKA_USER = "KafkaUser";
+    public static final String KAFKA_NODE_POOL = "KafkaNodePool";
+
     // ----------
     // Console
     // ----------
-    public static final String CONSOLE = "console";
+    public static final String CONSOLE = "Console";
```

Comment:
Do you have kind in the Console CR?

Reply:
Not that i am aware in ConsoleResource, no.

Reply:
@MikeEdgar would it make sense to add the kind here: https://github.com/streamshub/console/blob/main/operator/src/main/java/com/github/streamshub/console/api/v1alpha1/Console.java
?

I mean, the `kind` in the CR is taken from the class name? Usually I saw that these CR classes has the `KIND` or `RESOURCE_KIND` constant just to provide it to users that are also using the API. Or if it should be taken just from the display name of the class (IDK how it is handled in the generation)?

Reply:
Yeah it's just the class name - `Class#getSimpleName` would provide it.

Reply:
Fabric8 will also get it for you if you use `HasMetadata.getKind(Console.class)`.